src/connectors/quip.tinyspeck/mod.ts

```diff
@@ -0,0 +1,13 @@
+/** This file was autogenerated. Follow the steps in src/schema/slack/functions/README.md to rebuild **/
+import AddSpreadsheetRow from "./functions/add_spreadsheet_row.ts";
+
+const QuipTinyspeck = {
```

Comment:
Are we intentionally publishing a connector with `Tinyspeck` in the name?

Reply:
Nope I don't think we should do this... let me see whats up with this